DangoPlop/Assets/Scripts/PlayerController.cs

```diff
@@ -87,13 +107,64 @@ private void OnCollisionEnter2D(Collision2D collision)
         }
     }
 
+	public void Fire(){
+		if (bulletType == BulletType.DefaultFire) {
+			nextFire = Time.time + FireRate;
+			Instantiate (Projectile, ProjectilePos.position, Quaternion.identity);
+			Ammo--;
+		} 
+		else if (bulletType == BulletType.Laser) {
+			nextFire = Time.time + LaserRate;
+			Instantiate (Laser, ProjectilePos.position, Quaternion.identity);
+			Ammo--;
+		} 
+		else if (bulletType == BulletType.DoubleShot && currentDoubleShotAmmo > 0) {
+			nextFire = Time.time + DoubleShotRate;
+			Vector2 temp = ProjectilePos.transform.position;
+			Vector2 temp1 = ProjectilePos.transform.position;
+			var doubleShot1 = Instantiate (Projectile, ProjectilePos.position, Quaternion.identity);
+			var doubleShot2 = Instantiate (Projectile, ProjectilePos.position, Quaternion.identity);
+			doubleShot2.transform.position = temp;
+			doubleShot2.transform.Translate (FirstBulletTranslateX, FirstBulletTranslateY, 0, Space.World);
+			doubleShot1.transform.position = temp;
+			doubleShot1.transform.Translate (SecondBulletTranslateX, SecondBulletTranslateY, 0, Space.World);
+			currentDoubleShotAmmo--;
```

Comment:
Can you do this in a few lines of code. Seems like a lot of steps to get what we want when we should just set the new position based off of ProjectilePosition. Another developer shouldn't need to take time to understand how the double shot positioning works